applications/StructuralMechanicsApplication/python_scripts/structural_mechanics_implicit_dynamic_solver.py

```diff
@@ -91,14 +92,18 @@ def AddVariables(self):
         if self.settings["rotation_dofs"].GetBool():
             # Add specific variables for the problem (rotation dofs)
             self.main_model_part.AddNodalSolutionStepVariable(KratosMultiphysics.StructuralMechanicsApplication.POINT_TORQUE)
+        '''elif self.settings["multi_point_constraints_used"].GetBool():
+            self.main_model_part.AddNodalSolutionStepVariable(KratosMultiphysics.StructuralMechanicsApplication.IS_SLAVE)
```

Comment:
I would not add those values.

the SLAVE flag is already there,

the SLAVES list can be obtained by looping the nodes and seeing which ones are marked as SLAVE